### DISCUSSION TITLE
Nice Project!

### DISCUSSION THREAD

**Discussion by vurvdev** [revosw/wiremodforall#26] [General]
Writing this since I see that development has kinda stopped and the fact that I didn't know about it when it's existed for two years is pretty crazy to me, just want to say that this is amazing

If this got more attention maybe it could be endorsed as an alternative to the wiki, since that's very barebones right now, I also make a lot of stuff for wiremod so let me know if you need anything.

For example I recently made this https://vurv78.github.io/ExpressionScript which I plan to turn into backwards compatible E2 for the web / your pc, so if it got further enough along it could maybe be embedded into this as well

(Also I don't know if you've noticed my recent endeavors in changing E2 e.g. https://github.com/wiremod/wire/pull/2174, so this guide might need to be updated a little 😄)

**Comment by revosw** [comment]
Thanks for reaching out!

I'm going full steam on my third (and thankfully last) year in university, so I sadly cannot take much time out of my day to work on this website. What I was working on before stopping was to establish in what order should each e2 api be presented. I would be really grateful if you joined in on the project, and worked to make this guide more fleshed out. I'm mainly a website developer, so I can work on all the fancy animations and infrastructure

Let's chat on Discord, my name is Revolution#1234
